CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Changelog for NeoFS Node
 - Stored payload metric per shard (#2023)
 - Histogram metrics for RPC and engine operations (#2351) 
 - SN's version is announced via the attributes automatically but can be overwritten explicitly (#2455)
+- New `blobovnicza-to-peapod` tool providing blobovnicza-to-peapod data migration (#2453)
```

Comment:
I don't see an entry introducing `peapod` itself and warning that this is the last version to support bbcz.

Reply:
added